<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Individual</class>
 <widget class="QWidget" name="Individual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>600</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>  Назад</string>
   </property>
   <property name="icon">
    <iconset resource="qrs.qrc">
     <normaloff>:/left-arrow.svg</normaloff>:/left-arrow.svg</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>900</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Укажите параметры:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="qrs.qrc">
     <normaloff>:/renew.svg</normaloff>:/renew.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>600</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Подтвердить  </string>
   </property>
   <property name="icon">
    <iconset resource="qrs.qrc">
     <normaloff>:/tip.svg</normaloff>:/tip.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Максимальная комиссия на перевод  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>240</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Максимальная стоимость оповещения в месяц:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>210</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>в мобильном приложении</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>карты своего банка с карты другого</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>мобильной связи / Интернета</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>электронного кошелька</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>госуслуг / налогов / штрафов</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>по СМС</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
   </property>
   <item>
    <property name="text">
     <string>между своими картами и счетами</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>клиенту этого же банка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>клиенту другого банка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>через систему быстрых платежей</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>320</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Максимальная комиссия на пополнение / оплату  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>330</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qrs.qrc"/>
 </resources>
 <connections/>
</ui>
